test/apply-legacy/specs/index.js

```diff
@@ -22,33 +22,43 @@ const waitForSync = browser => {
   });
 };
 
-const addProtocol = browser => {
-  browser.$('input[name$=".title"]').setValue(words(6));
-  browser.$('.protocol').click('button=Continue');
-  browser.setValue('input[name$=".severity"]', sample(['Mild', 'Moderate', 'Severe', 'Non-recovery']));
+const addProtocol = (browser, title) => {
+  if (!browser.$('input[name$=".title"]').isVisible()) {
+    console.log('Adding another protocol');
+    browser.click('button=Add another protocol');
+    browser.waitForExist('input[name$=".title"]');
+  }
 
-  browser
+  browser.scroll(0, 5000);
+
+  browser.$('input[name$=".title"]').setValue(title);
+  browser.$('.protocol:last-of-type button').click();
```

Comment:
Can we be more specific than just `button`? Purely for my benefit trying to follow what's going on?